site/docs/advanced/reliability.md

```diff
@@ -146,26 +146,37 @@ The remainder of this section is elaborating on **how** grammY behaves under the
 
 > Do you just care about coding a Telegram bot? [Skip the rest of this page.](./flood.md)
 
-If you are running your bot on webhooks, the Bot API server will retry delivering updates to your bot if it does not respond with OK in time.
+### Webhook
+
+If you are running your bot on webhooks, the Bot API server will retry delivering updates to your bot if it does not respond with `OK` in time.
 That pretty much defines the behavior of the system comprehensively—if you need to prevent processing duplicate updates, you should build your own de-duplication based on `update_id`.
 grammY does not do this for you, but feel free to PR if you think someone else could profit from this.
 
+### Long Polling
+
 Long polling is more interesting.
 The built-in polling basically re-runs the most recent update batch that was fetched but could not complete.
-(Note that if you properly stop your bot with `bot.stop`, the update offset will be synced with the Telegram servers by calling `getUpdates` with the correct offset but without processing the update data).
+
+> Note that if you properly stop your bot with `bot.stop`, [the update offset](https://core.telegram.org/bots/api#getting-updates) will be synced with the Telegram servers by calling `getUpdates` with the correct offset but without processing the update data.
+
 In other words, you will never loose any updates, however, it may happen that you re-process up to 100 updates that you have seen before.
 As calls to `sendMessage` are not idempotent, users may receive duplicate messages from your bot.
 However, _at least once_ processing is guaranteed.
 
+### grammY Runner
+
 If you are using the [grammY runner](../plugins/runner.md) in concurrent mode, the next `getUpdates` call is potentially performed before your middleware processes the first update of the current batch.
-Thus, the update offset is [confirmed](https://core.telegram.org/bots/api#getupdates) prematurely.
+Thus, the [update offset](https://core.telegram.org/bots/api#getupdates) is confirmed prematurely.
```

Comment:
Please revert. This refers to the process of confirming updates, rather than the update offset itself.